activeEvents.js

```diff
@@ -0,0 +1,34 @@
+var description = null;
+var clearButton = document.querySelector('#clear')
```

Comment:
Not sure why you're declaring these vars outside of the IIFE function. The point of an IIFE is to reduce the number of global variables and to control what is private vs public.